indico/core/celery/templates/celery/celery_tasks.html

```diff
@@ -0,0 +1,61 @@
+{% from 'message_box.html' import message_box %}
+{% extends 'layout/base.html' %}
+
+{% block title %}
+    {% trans %}Tasks{% endtrans %}
+{% endblock %}
+
+{% block content %}
+    <div class="tasks">
+        {%- call message_box('info', fixed_width=True) -%}
+            {%- if flower_url -%}
+                {%- trans -%}Monitor Indico related tasks with flower <a class="bold" href="{{ flower_url }}" target="_blank">here</a>.{%- endtrans %}
+            {%- else -%}
+                {%- trans -%}
+                    Indico uses Celery to manage tasks, which can be monitored using Flower.<br>
+                    Configure the Flower URL in <span class="mono">indico.conf</span> to have a link displayed here.
+                {%- endtrans %}
+            {%- endif -%}
+        {%- endcall %}
+        <h2>Periodic Tasks</h2>
+        <table class="i-table-widget fixed-width">
+            <thead>
+                <tr>
+                    <th>{% trans %}Name{% endtrans %}</th>
+                    <th>{% trans %}Schedule{% endtrans %}</th>
+                    <th></th>
+                </tr>
+            </thead>
+            <tbody>
+                {%- for task in tasks %}
+                    <tr>
+                        <td>{{ task.name }}</td>
+                        <td>{{ render_task_schedule(task) }}</td>
+                        <td>{{ render_task_label(task) }}</td>
+                    </tr>
+                {%- endfor %}
+            </tbody>
+        </table>
+    </div>
+{% endblock %}
+
+{% macro render_task_schedule(task) -%}
+    {%- if not task.disabled -%}
```

Comment:
wouldn't that be better off outside? i.e. "if not task.disabled: render_task_schedule()"